### DISCUSSION TITLE
419 page expired

### DISCUSSION THREAD

**Discussion by zeyadsharo** [laravel/nova-issues#3407] [Q&A]
- Laravel Version:  8
- Nova Version: v3.14.0
- PHP Version: ^7.3|^8.0
- Database Driver & Version: MySQL  workbench 8.0
- Operating System and Version: Window 10
- Browser type and version: chrome, edge, Tor 

### Description:
in local it login normally work 
but when I deploy it to server shared host 
login has not worked just when I click 419  page expired

**Comment by crynobone** [comment]
I would start by checking session driver used on production environment. Laravel Nova simply utilizes Laravel Framework session and we don't do anything custom for session.

**Comment by Andrewrafat** [comment]
so do nay one found solution
